app/presenters/document_export_presenter.rb

```diff
@@ -46,6 +47,13 @@ def complete_images_hash(edition_images)
     end
   end
 
+  def provide_doctype_information(edition, output)
+    output[:document_type_information] = edition.public_send(:news_article_type).as_json if edition.respond_to?(:news_article_type)
+    output[:document_type_information] = edition.public_send(:publication_type).as_json if edition.respond_to?(:publication_type)
+    output[:document_type_information] = edition.public_send(:corporate_information_page_type).as_json if edition.respond_to?(:corporate_information_page_type)
+    output[:document_type_information] = edition.public_send(:speech_type).as_json if edition.respond_to?(:speech_type)
```

Comment:
I think it'd be more intuitive for these to have a key that reflects their name - as these aren't actually document type information (which is a field on document).

So I'd have these set as `:news_article_type`, `publication_type`, etc.

Reply:
This makes more sense when providing all the type data, however when providing the key only as discussed above then yes I'll be tidying these up.